notebooks/official/generative_ai/text_embeddings.ipynb

```diff
@@ -0,0 +1,430 @@
+{
```

Comment:
<p>The Github link is wrong.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/3672/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Done: Commit #4